### ISSUE TITLE
TypeError: unsupported operand type(s) for |=: 'OrderedSet' and 'OrderedSet'

### ISSUE THREAD

**Issue by MattMuffin** [JelteF/PyLaTeX#293]
Hello,

I have this error on a fresh install.

```
user_admin@VM:~$ pip3 install pylatex
Collecting pylatex
  Using cached PyLaTeX-1.3.3.tar.gz (83 kB)
Requirement already satisfied: ordered-set in /usr/local/lib/python3.5/dist-packages (from pylatex) (4.0.2)
Building wheels for collected packages: pylatex
  Building wheel for pylatex (setup.py) ... done
  Created wheel for pylatex: filename=PyLaTeX-1.3.3-py3-none-any.whl size=44334 sha256=0a499ec4c379dee1e68f3dedf35cbef82bdfc7a6d685ea8f631c2c4ebb819aaf
  Stored in directory: /home/user_admin/.cache/pip/wheels/75/18/f4/a085874287a46bf33f01629f372840d8045c5e95e5c85d1543
Successfully built pylatex
Installing collected packages: pylatex
**Successfully installed pylatex-1.3.3**
```
And testing:

```
user_admin@VM:~$ python3
**Python 3.5.2** (default, Jul 17 2020, 14:04:10) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pylatex
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/dist-packages/pylatex/__init__.py", line 8, in <module>
    from .basic import HugeText, NewPage, LineBreak, NewLine, HFill, LargeText, \
  File "/usr/local/lib/python3.5/dist-packages/pylatex/basic.py", line 9, in <module>
    from .base_classes import CommandBase, Environment, ContainerCommand
  File "/usr/local/lib/python3.5/dist-packages/pylatex/base_classes/__init__.py", line 9, in <module>
    from .containers import Container, Environment, ContainerCommand
  File "/usr/local/lib/python3.5/dist-packages/pylatex/base_classes/containers.py", line 13, in <module>
    from .command import Command, Arguments
  File "/usr/local/lib/python3.5/dist-packages/pylatex/base_classes/command.py", line 18, in <module>
    class CommandBase(LatexObject):
  File "/usr/local/lib/python3.5/dist-packages/pylatex/base_classes/latex_object.py", line 22, in __init__
    packages |= b.packages
TypeError: unsupported operand type(s) for |=: 'OrderedSet' and 'OrderedSet'
```

I have tried reinstalling from scratch, same result.

Using Ubuntu 16.04

Thank you

**Comment by JelteF** [comment]
Sounds like the OrderedSet library broke its api. Downgrading probably
fixes it.

**Comment by MattMuffin** [comment]
Dowgrading OrderedSet?

**Comment by MattMuffin** [comment]
Just dowgraded to ordered-set 3.1.1 seems to have worked. Thanks for the hint!

**Comment by JelteF** [comment]
could you report this as an issue to the ordered-set library github?

**Comment by JelteF** [comment]
I just released pylatex 1.3.4 that uses ordered-set<4.0.0 for python 3.5 or lower.
